src/Microsoft.Data.Analysis.Interactive/DataFrameKernelExtension.cs

```diff
@@ -38,12 +98,27 @@ public Task OnLoadAsync(IKernel kernel)
 
             directive.AddOption(new Option<string>(
                 "typeName",
-                getDefaultValue:() => "Foo"));
+                getDefaultValue: () => "Foo"));
 
             kernelBase.AddDirective(directive);
-            
+
             return Task.CompletedTask;
 
+            static string BuildPageScript(int page, int size, string uniqueId)
+            {
+                var script = string.Empty;
+                script += $"var els = document.querySelectorAll('#{uniqueId} tbody tr:nth-child(n+{page * size + 1})'); ";
```

Comment:
What does `els` stand for? What do you think about spelling it out?

Reply:
unfortunate naming here, a contracted form for elements, I will rename to rows.